tensorboard/plugins/graph/tf_graph_controls/tf-graph-controls.ts

```diff
@@ -1131,6 +1130,8 @@ export class TfGraphControls extends LegacyElementMixin(PolymerElement) {
   })
   _legendOpened: boolean = true;
 
+  private _downloadFilename = '';
```

Comment:
Should this default to `'graph.png'` for consistency with the setter?

Reply:
Done.